### PR TITLE
Add VSCode in the Browser editing hints

### PR DESCRIPTION
**Pull request by enryH**
During the tutorial/workshop on the 20.04.2022, I added some hints to VS Code in the Browser.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,12 +42,30 @@ Without any prior setup on your local site, you can edit files directly on githu
...
 
-> you commit your changes to the branch you selected!
+> you commit (=save) your changes to the branch you selected!
+
+### VS Code in the browser editing 
+
+Instead of editing in the browser using the GitHub user interface, you can also directly use
+the editor VSCode in the browser:
+
+- In [github.com/cphbiosciencephd/info..._pages/bikes.md)](https://github.com/cphbiosciencephd/info/blob/master/_pages/bikes.md) exchange the `.com` for `.dev`, i.e. [github.dev/cphbiosciencephd/info..._pages/bikes.md)](https://github.dev/cphbiosciencephd/info/blob/master/_pages/bikes.md)
```
Comment (ri-heme) [inline README.md]: There's an extra `)` in both links.

**Review by ri-heme — CHANGES_REQUESTED** [review]
Cool ✨ ! I didn't know about the browser editing feature 👍🏽

File: README.md (hunk 1 above)
Comment (enryH) [reply README.md]: thanks for spotting this:) You can also directly edit it in the form:)

Comment (enryH) [inline README.md]:
```suggestion
- In [github.com/cphbiosciencephd/info..._pages/bikes.md](https://github.com/cphbiosciencephd/info/blob/master/_pages/bikes.md) exchange the `.com` for `.dev`, i.e. [github.dev/cphbiosciencephd/info..._pages/bikes.md](https://github.dev/cphbiosciencephd/info/blob/master/_pages/bikes.md) 
```

Comment (enryH) [reply README.md]: it's the first icon (with the +/- sign) which says "Add a suggestions":)

**Review by ri-heme — APPROVED** [review]

**Pull request merged**